assets/src/parse_bundle_plugin.js

```diff
@@ -8,14 +9,15 @@ var BundleTracker = require('webpack-bundle-tracker');
 var fs = require("fs");
 var path = require("path");
 var logging = require('./logging');
+var webpack = require('webpack');
+var base_config = require('../../webpack.config.base');
```

Comment:
Trying to avoid relative paths as much as possible (hence the 'component' structure I have urged in the Frontend Docs - where code that will be imported together should be in the same directory) but in this case, I think a relative path is unavoidable unless we define an alias for node somewhere else.